### PR TITLE
Fix typo in README.md

### PR DESCRIPTION
**Pull request by yeonhoyoon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -909,7 +909,7 @@ config.wrappers do |b|
 end
 ```
 
-This you set the input and label class to `'label-input-class'` and will set the class `'is-invalid'`
+This sets the input and label class to `'label-input-class'` and will set the class `'is-invalid'`
```
Comment (tegon) [inline README.md]:
```suggestion
This sets the input and label classes to `'label-input-class'` and will set the class `'is-invalid'`
```

File: README.md [hunk 2]
```diff
@@ -909,7 +909,7 @@ config.wrappers do |b|
...
 when the input has errors and `'is-valid'` if the input is valid.
```
Comment (tegon) [inline README.md]:
```suggestion
if the input has errors and `'is-valid'` if the input is valid.
```

**Comment by tegon** [comment]
Thanks! I just added a small suggestion, let me know what you think about it.

**Comment by yeonhoyoon** [comment]
Thanks for taking a close look! I’ve applied your suggestions.

**Comment by tegon** [comment]
Thanks!

**Pull request merged**